### ISSUE TITLE
Give site owners ability to resend new sale email notification

### ISSUE THREAD

**Issue by ciorici** [easydigitaldownloads/easy-digital-downloads#6557]
Hello,

Here's is my feature request or idea for future versions: **Resend New Sale Email Notification to Admin**

I'll be short by adding a screenshot of what I would like to see when viewing a Payment details:

![edd-resend](https://user-images.githubusercontent.com/1674706/39969256-6acd7132-56d9-11e8-8f80-2b874a937004.png)

Sometimes due to some server problems it's possible to miss a New Sale notification sent to site admin, and I couldn't find a way to resend it, as I like to keep a copy of everything in my email, so I could easily search by a customer name or email address and find all his previous purchases much faster.

Basically, I would find really helpful something like Resend Receipt extension, but for site owners, or maybe you can update this extension in the future and include this option in it.

Thank you

**Comment by SDavisMedia** [comment]
Hi @ciorici. This isn't a bad idea at all. We like the idea of this being a feature of Resend Receipt. So I'm going to log an issue over on that repo to make the enhancement.

Thanks for the suggestion!
